netci.groovy

```diff
@@ -15,11 +15,16 @@ def static getBuildJobName(def configuration, def os) {
 }
 
 [true, false].each { isPullRequest ->
-    ['Debug', 'Release'].each { configuration ->
+    ['Debug', 'ci_part1', 'ci_part2'].each { configuration ->
```

Comment:
it's possible to do another `['ci_part1', 'ci_part2'].each { ci_part ->`  so `Debug` is not different from `Release`